README.md

```diff
@@ -1,3 +1,5 @@
+
+[![@etix](https://avatars0.githubusercontent.com/u/38853?s=120&v=4)
```

Comment:
I'm not interested in having my avatar on the README.

Reply:
Ok thought it be very nice to put a face on all the hard work you do.